docs/io.cozy.files.md

```diff
@@ -50,10 +50,16 @@ The attributes of a file are:
     - `x`: {float}: x coordinate in the photo where the person is
     - `y`: {float}: y coordinate in the photo where the person is
   - `icon`: {string} : A small SVG file to use as icon for this file (optional)
+  - `carbonCopy`: {boolean} : if the document is the original document imported by the connector
```

Comment:
There is no `carbonCopy_by` ?

Reply:
nope, it would be the same as `cozyMetadata.uploadedBy`